gp-easy-passthrough/gpep-edit-entry.php

```diff
@@ -246,6 +249,24 @@ public function exclude_edit_entry_from_inventory( $query, $field ) {
 		return $query;
 	}
 
+	public static function purge_product_cache( $form, $entry ) {
+
+		$cache_options = array(
+			array( false, false ),
+			array( false, true ),
+			array( true, false ),
+			array( true, true ),
+		);
+
+		foreach ( $cache_options as $cache_option ) {
+			list( $use_choice_text, $use_admin_label ) = $cache_option;
+			if ( gform_get_meta( rgar( $entry, 'id' ), "gform_product_info_{$use_choice_text}_{$use_admin_label}" ) ) {
+				gform_delete_meta( rgar( $entry, 'id' ), "gform_product_info_{$use_choice_text}_{$use_admin_label}" );
+			}
+		}
+
+	}
```

Comment:
This looks super similar `GFFormsModel::refresh_product_cache()`. What led you to duplicate it rather calling it directly?

Reply:
@claygriffiths  the `GFFormsModel::refresh_product_cache()` is calling the `GFCommon::get_product_fields` [here](https://github.com/gravityforms/gravityforms/blob/master/forms_model.php#L3504) and the issue persist but the updated value will show up on further edit.

Reply:
Got it! Thank you